Dockerfile

```diff
@@ -4,13 +4,15 @@ COPY . .
 RUN make
 
 FROM registry.svc.ci.openshift.org/openshift/origin-v4.0:base
+ARG CSV=4.1
 
 ENV ALERTS_FILE_PATH="/etc/elasticsearch-operator/files/prometheus_alerts.yml"
 ENV RULES_FILE_PATH="/etc/elasticsearch-operator/files/prometheus_rules.yml"
 
 COPY --from=builder /go/src/github.com/openshift/elasticsearch-operator/_output/bin/elasticsearch-operator /usr/bin/
 COPY files/ /etc/elasticsearch-operator/files/
-ADD controller-manifests /manifests
+COPY /manifests/elasticsearch-operator.package.yaml /manifests
```

Comment:
I think the source of the copy should not be an absolute path - try `manifests/` instead of `/manifests`